src/components/Resource/ResourceCreate.tsx

```diff
@@ -145,23 +152,34 @@ export default function ResourceCreate(props: ResourceProps) {
     <Page
       title={t("create_resource_request")}
       crumbsReplacements={{
-        [facilityId]: { name: facilityData?.name || "" },
+        [facilityId]: {
+          name: facilityData?.name || "",
+          uri: `/facility/${facilityId}/settings/general`,
+        },
         resource: { style: "pointer-events-none" },
       }}
-      backUrl={`/facility/${facilityId}`}
+      backUrl={`/facility/${facilityId}/settings/general`}
     >
       <div className="container mx-auto max-w-4xl">
         <Card className="mt-4">
           <Form {...form}>
             <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-6">
-              {related_patient && (
+              {patientData && (
                 <Alert>
                   <div className="flex items-center gap-2">
-                    <CareIcon icon="l-user" className="h-5 w-5 text-blue-700" />
-                    <AlertDescription className="text-sm text-blue-700">
-                      {t("linked_patient")}:{" "}
-                      <span className="font-medium">{related_patient}</span>
-                    </AlertDescription>
+                    <Link
+                      href={`/facility/${facilityId}/patient/${related_patient}/demography`}
```

Comment:
Let's link to resources tab instead.

```suggestion
                      href={`/facility/${facilityId}/patient/${related_patient}/resource_requests`}
```